frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -1,10 +1,11 @@
 <script setup lang="ts">
-import Dropdown, { type DropdownChangeEvent } from 'primevue/dropdown';
 import { onUnmounted, shallowRef, type PropType } from 'vue';
-
 import ManagePermissionsTitle from '@/components/managePermissions/ManagePermissionsTitle.vue';
 import UserDataTable from '@/components/managePermissions/UserDataTable.vue';
-import { Severity } from '@/enum/SeverityEnum';
+import Dropdown, { type DropdownChangeEvent } from 'primevue/dropdown';
+import TabView from 'primevue/tabview';
+import TabPanel from 'primevue/tabpanel';
```

Comment:
as we mentioned before, import from external package is better before the local import, just good practice.

Reply:
sure, I'll fix it